app/src/main/java/org/jellyfin/mobile/utils/HtmlUtils.kt

```diff
@@ -0,0 +1,12 @@
+package org.jellyfin.mobile.utils
+
+import android.text.Html
+
+@Suppress("DEPRECATION")
+fun stripHtmlChars(content: String?): String {
+    return if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
+        Html.fromHtml(content, Html.FROM_HTML_MODE_LEGACY).toString()
+    } else {
+        Html.fromHtml(content).toString()
+    }
+}
```

Comment:
You should be able to replace this function with [`HtmlCompat.fromHtml()`](
https://developer.android.com/reference/androidx/core/text/HtmlCompat#fromHtml(java.lang.String,int)) called directly in `RemotePlayerService`. Note that the passed content string must be non-null, thus you should wrap the `fromHtml` call with a let, e.g. `title?.let { HtmlCompat.fromHtml(it, Html.FROM_HTML_MODE_LEGACY) }`.

Reply:
K, I made the fix as requested. For my edification, what happens if the null check returns true? Does the `it` lambda parameter fall through to null?

Reply:
If `title` is null, the let won't be executed and the whole statement evaluates to null.